users/ignoring.rst

```diff
@@ -140,3 +142,47 @@ all files and directories called "foo", ending in a "2" or starting with
   ``some/directory/`` matches the content of the directory, but not the
   directory itself. If you want the pattern to match the directory and its
   content, make sure it does not have a ``/`` at the end of the pattern.
+
+Ignoring files across all devices
+---------------------------------
+
+Although ``.stignore`` file is not synced between the devices, as mentioned
+previously, it's possible to take an advantage of ``#include`` pattern
+to create a single file that will hold globally ignored patterns.
+
+Assuming the content of the file is::
+
+    // Globally ignored patterns
+    (?d).DS_Store
+    foo
+
+This file can be synced across all devices and helps with ignoring
+``.DS_Store`` if you sync a folder between devices where at least one is
+Mac OS and will also cause ``foo`` pattern being ignored on all devices.
+
+.. note::
+   This way in case your syncing gets "stuck" with displaying some
+   pending unexpected files and you can't reach the device that contains
+   those files, simply add it to ``.stglobalignore``.
+
+The important part is to ``#include`` this global ignore file per each
+device so that::
+
+    // .stignore for 1st Device
+    #include .stglobalignore
+
+    // .stignore for 2nd Device
+    #include .stglobalignore
+
+    ...
+
+    // .stignore for N-th Device
+    #include .stglobalignore
+
+The global ``#include`` does not exclude the possibility to add custom
+patterns per each device.
+
+.. note::
+   This can be added even later with a lot of devices already being present
+   in Syncthing, but it's a way easier to set up ignoring in the beginning
+   than to include that global file to each device separately.
```

Comment:
Why, what does this help the user? I mean of course if you ignore from the get-go, you save some hashing/indexing, but that's got nothing to do with shared ignores, does it?

Reply:
It's rather about the setting up of a single shared file because otherwise you'll pull the folder data first and if you have e.g. `.DS_Store` on Mac **and** you pull the data to Mac, that OS will pre-create `.DS_Store` if you attempt to open it in `Finder` and will result in Mac trying to sync `.DS_Store` files to a node that has such files listed as ignored.

If you want, I will remove it.